clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/validators/StandardAppEngineAttributeValidator.groovy

```diff
@@ -233,4 +236,32 @@ class StandardAppEngineAttributeValidator {
     }
     return valid
   }
+
+  def validateServingStatusCanBeChanged(String serverGroupName,
+                                        AppEngineNamedAccountCredentials credentials,
+                                        AppEngineClusterProvider clusterProvider,
+                                        String attribute) {
+    def valid = true
```

Comment:
This routine might read easier if you dropped the use of `valid` entirely and just return `true` or `false` directly in each spot.